video_creation/final_video.py

```diff
@@ -54,7 +54,7 @@ def make_final_video(number_of_clips):
     image_concat.audio = audio_composite
     final = CompositeVideoClip([background_clip, image_concat])
     final.write_videofile(
-        "assets/final_video.mp4", fps=30, audio_codec="aac", audio_bitrate="192k"
+        "output/" + name + ".mp4", fps=30, audio_codec="aac", audio_bitrate="64k"
```

Comment:
why the lowering of the audio bitrate?

Reply:
To reduce file sizes and time to write file.